about/changelog.md

```diff
@@ -0,0 +1,147 @@
+
+To access the compression wizard, navigate to `Explorer`, and select the hypertable you would like to compress. In the top right corner, hover where it says `Compression off`, and open the wizard. You will then be guided through the process of configuring compression for your hypertable, and can compress it directly through the UI.
+
+![Run the compression wizard in Timescale Console](https://assets.timescale.com/docs/images/compress-data-in-console.png)
+
+## 🏎️💨 High Performance AI Apps With pgvectorscale
+
+<Label type="date">June 11, 2024</Label>
+
+The [vectorscale extension][pgvectorscale] is now available on [Timescale Cloud][signup].
+
+pgvectorscale complements pgvector, the open-source vector data extension for PostgreSQL, and introduces the
+following key innovations for pgvector data:
+
+- A new index type called StreamingDiskANN, inspired by the DiskANN algorithm, based on research from Microsoft.
+- Statistical Binary Quantization: developed by Timescale researchers, This compression method improves on
+  standard Binary Quantization.
+
+On benchmark dataset of 50 million Cohere embeddings (768 dimensions each), PostgreSQL with pgvector and
+pgvectorscale achieves 28x lower p95 latency and 16x higher query throughput compared to Pinecone's storage
+optimized (s1) index for approximate nearest neighbor queries at 99% recall, all at 75% less cost when
+self-hosted on AWS EC2.
+
+See the [pgvectorscale documentation][pgvectorscale] to learn more.
+
+## 🧐 Integrate AI Into Your Database Using pgai
+
+<Label type="date">June 11, 2024</Label>
+
+The [pgai extension][pgai] is now available on [Timescale Cloud][signup].
+
+pgai brings embedding and generation AI models closer to the database. With pgai, you can now do the following directly
+from within PostgreSQL in a SQL query:
+
+* Create embeddings for your data.
+* Retrieve LLM chat completions from models like OpenAI GPT4o.
+* Reason over your data and facilitate use cases like classification, summarization, and data enrichment on your existing relational data in PostgreSQL.
+
+See the [pgai documentation][pgai] to learn more.
+
+
+## TimescaleDB 2.15.2
+<Label type="date">June 7, 2024</Label>
+
+This release contains bug fixes introduced since TimescaleDB v2.15.1.
+Best practice is to upgrade at the next available opportunity.
+
+- [#6975](https://github.com/timescale/timescaledb/issues/6975): Fix sort pushdown for partially compressed chunks.
+- [#6976](https://github.com/timescale/timescaledb/issues/6976): Fix removal of the metadata function and the update script.
+- [#6978](https://github.com/timescale/timescaledb/issues/6978): Fix segfault in compress_chunk with primary space partition.
+- [#6993](https://github.com/timescale/timescaledb/issues/6993): Disallow hash partitioning on the primary column.
+
+Want to know more? See the [release notes][timescaledb-releases].
+
+## May 2024 Updates
```

Comment:
What really works in the UI (https://docs-dev.timescale.com/docs-changelog-ab-test/about/changelog-ab-test/changelog/) is having the list of new features in the TOC on the right, especially with the cool emojis. It will make people smile before they start reading.  Could you please use your structure that enables this throughout:

## 😎 Feature title
date

Description

Link to release note or repo.

Reply:
Agree. Feature titles or date titles, but I don't think a mix works